apps/keyboard/js/layouts/th.js

```diff
@@ -0,0 +1,85 @@
+Keyboards.th = {
+  label: 'Thai',
+  shortLabel: 'Th',
+  menuLabel: 'ไทย', /*ไทย*/
+  basicLayoutKey: 'กขค', /*กขค*/
+  imEngine: 'latin',
```

Comment:
Yes, you are correct. Thai is not a latin language. I had left it there from the previous patch. Looking through the code I see that there is a default IME engine, which is fine for this.